### DISCUSSION TITLE
Add logic to have ReservedPaths maintain the default values

### DISCUSSION THREAD

**Discussion by LennardF1989** [umbraco/Umbraco-CMS#14108] [Features and ideas]
Right now it's possible to set ReservedPaths from appsettings.json, but this effectively overrides to default value. Most of the time you do not want to override but instead add to it.

This can be solved by introducing another config-key for "AdditionalReservedPaths" that get appended to the default value. Another solution is making a special string like %default%, that can be included in the ReservedPaths like so:
```json
{
    "ReservedPaths": "%default%,~/myadditionalpath"
}
```

Right now, you have to always make sure the default value is up-to-date if you override it.

**Comment by bergmania** [comment]
IMO it would be more clean to have an `AdditionalReservedPaths`.

Not sure how big an issue it really is. Do you often have different `ReservedPaths` per environment? Otherwise I would add those in code instead.

**Comment by LennardF1989** [comment]
It's occasional use, but doing it in code is not handy because it's usually to expose static files without Umbraco mixing in and those get shuffled around. So I'll just end up adding my own appsettings config key then :P
